src/combat/-read-.-this-

```diff
@@ -0,0 +1 @@
+Directory for Muddle COMBAT plan files.
```

Comment:
There already is a doc/combat/-read-.-this- so I'll remove this copy.

Reply:
Ok